news/2017-05-05-weld-300Final.asciidoc

```diff
@@ -0,0 +1,70 @@
+---
+layout: news
+title: Weld 3.0.0.Final - the first implementation of CDI 2.0!
+author: Martin Kouba
+priority: 1
+change_frequency: weekly
+excerpt: Weld 3 finally comes to the party. Try it out!
+desc: Weld 3.0.0.Finall released. Try out the first implementation of the CDI 2.0 spec!
+tags: [release]
+---
+:linkattrs:
+
+I am very pleased to announce the release of Weld *3.0.0.Final* - the first implementation of *CDI 2.0*!
+I would like to thank not only to everyone involved in this particular release but also to the Weld community as a whole and also to all active CDI EG members who pushed a lot of energy into the specification process!
+
+[ source, java ]
+----
+/**
+ * TODO: Deliver bugfixes and improvements!
```

Comment:
That's an intentional TODO - we as the Weld team promise to continue with development ;-)